gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/templates/alert_HTTPStatus.html

```diff
@@ -9,6 +9,7 @@
           ${alert.conditions[0].comparison.thresholdLow[0]}XX over ${alert.conditions[0].threshold}% during ${alert.conditions[0].duration} ${alert.conditions[0].timeUnit}.
         </#if>
       </p>
+      <p>${alert.description}</p>
```

Comment:
Description is not mandatory so, if not set, it will result an empty paragraph.

Should we better use a if statement ?

Reply:
👍